customizations/README.md

```diff
@@ -1,122 +1,96 @@
-# Customizations
+# Configurations
```

Comment:
In Helm/Chart/K8s this is called `customization` because you adapt your package: https://helm.sh/docs/intro/using_helm/#customizing-the-chart-before-installing

If you would like to proceed with the non helm standards, please, rename also the folder and all the necessary paths to avoid confusion.

Reply:
https://cartoteam.slack.com/archives/C02K9JU7UKF/p1648450151287389?thread_ts=1648225797.414389&cid=C02K9JU7UKF